Misc/NEWS.d/next/macOS/2019-07-13-15-58-18.bpo-18049.MklhQQ.rst

```diff
@@ -0,0 +1,2 @@
+Increase stack size of threads to 16MB on macOS, to match the stack size of
```

Comment:
Would you mind to mention the old value, 5 MiB?

You may specify that it's the *default* size: it can be overriden by _thread.stack_size(size).